src/views/Header.js

```diff
@@ -99,11 +80,63 @@ class Header extends React.Component<DefaultProps, HeaderProps, *> {
     );
   }
 
+  _getHeaderTitle(navigation: Navigation): ?string {
+    const header = this.props.router.getScreenConfig(navigation, 'header');
+    let title;
+    if (header && header.title) {
+      title = header.title;
+    } else {
+      title = this.props.router.getScreenConfig(navigation, 'title');
+    }
+    if (typeof title === 'string') {
+      return title;
+    }
+    return undefined;
+  }
+
+  _getHeaderTintColor(navigation: Navigation): ?string {
+    const header = this.props.router.getScreenConfig(navigation, 'header');
+    if (header && header.tintColor) {
+      return header.tintColor;
+    }
+    return undefined;
+  }
+
+  _renderTitleComponent = (props: SubViewProps) => {
+    const color = this._getHeaderTintColor(props.navigation);
+    const title = this._getHeaderTitle(props.navigation);
+    return <HeaderTitle style={{ color }}>{title}</HeaderTitle>;
```

Comment:
From testing, it doesn't override the default value, however I do see the point of keeping it explicit.